### PR TITLE
Add styled sidebar 

### PR DESCRIPTION
**Pull request by 0xStarcat**
This PR adds the sidebar styled to the specifications of [this mockup](https://github.com/paulvill/paulvill.github.io/files/1043046/eda_sitemocks-base.pdf). Please review!

The sidebar is made with pure CSS/HTML.

Features:
- Collapsable sections 
- Moving sliders
- Checkboxes 
- Gauges/Bars 
- And more

Each control and indicator is tagged with an ID attribute for the later work of hooking up the sidebar to the necessary functionality. As of now, there is no functionality programmed into the sidebar - only appearances. Naturally, it will need a few tweaks once the more of the page gets completed to stylistically integrate it with the rest of the page.

### CODE REVIEW

File: layout_test/assets/css/sidebar.css [hunk 1]
```diff
@@ -0,0 +1,393 @@
+.sidebar {
```
Comment (0xStarcat) [inline layout_test/assets/css/sidebar.css:1]: All the CSS properties are name-spaced to the `.sidebar` class to avoid collisions.

Comment (0xStarcat) [reply layout_test/assets/css/sidebar.css:1]: Colors were obtained from using a 'color-picker' tool on the mocks.

**Comment by paulvill** [comment]
Hi Jeff @0xStarcat  

Looks great! Thank you for following up! 

Thank you for ID attributes, I will work on having it communicate to the viewport. I might edit a little bit depending on the controls we end up using, but this is a great basis!

Paul

**Comment by defenestrated** [comment]
looks good, guys! @paulvill i agree i think we'll need to tweak slightly when we're ready to integrate it, but it's a great start – thanks so much, @0xStarcat!

**Pull request merged**